koji/playbooks/external_repos.yml

```
@@ -1,13 +1,33 @@
 ---
 - hosts: localhost
   gather_facts: false
+  collections:
+    - ktdreyer.koji_ansible
   vars_files:
     - ../vars/external_repos.yml
   tasks:
     - name: Configure Koji External Repositories
-      ktdreyer.koji_ansible.koji_external_repo:
+      koji_external_repo:
         name: "{{ item.name }}"
         url: "{{ item.url }}"
         state: "{{ item.state | default('present') }}"
         koji: 'koji'
       loop: "{{ external_repos }}"
+
+    - name: list of koji external repos from koji
+      koji_call:
+        name: listExternalRepos
+        koji: 'koji'
+      register: call_result
```

```suggestion
      register: external_repos_result
```

speaking names etc ;)